diagnostic.md

```diff
@@ -14,7 +18,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$("OL").children()
+$("OL").find("li")
```

Comment:
I don't think this would work with capital `OL` I usually see them as lowercase.